### DISCUSSION TITLE
gemmi has no (wrap_to_zero) anymore?

### DISCUSSION THREAD

**Discussion by lobnaOmar20** [project-gemmi/gemmi#333] [Q&A] [answered]
I was running a code I created with gemmi to calculate interatomic distance and it was running fine up to last month. Today it gave me this error 

`AttributeError: 'gemmi.Fractional' object has no attribute 'wrap_to_zero'
`

is this now deprecated or what?? 

Thanks in advance,
Lobna.

**Comment by wojdyr** [comment]
No, it hasn't changed. Perhaps you are using a much older gemmi version now?

**Reply by lobnaOmar20** [reply]
I am using 0.5.7.
I did the code one year ago and since I use it over and over again, it was working fine up to one month ago. I didn't change anything.

**Reply by wojdyr** [reply, marked as answer]
If a function is deprecated or removed in gemmi (this one wasn't), it doesn't affect you until you update the version installed on your computer. Ver. 0.5.7 was released 2 years ago. I suspect a month ago you used a newer version.

**Reply by lobnaOmar20** [reply]
I will try to update and let you know if it didn't work out. 
But how you say I used a newer version and then now it is the older version??

**Reply by wojdyr** [reply]
wrap_to_zero was added in v0.6.0

**Reply by lobnaOmar20** [reply]
yess it worked!! Thank you very much :) 
By the way, I mentioned gemmi library in my latest paper and mentioned you personally in the acknowledgments for your quick, kind support.
